src/table/table.stories.ts

```diff
@@ -37,8 +38,10 @@ import { clone } from "../utils/utils";
 			[model]="model"
 			[size]="size"
 			[showSelectionColumn]="showSelectionColumn"
+			[enableSingleSelect]="enableSingleSelect"
```

Comment:
This is actually not officially supported by current carbon design so we're not going to surface it in the story.

Reply:
Sure, didn't know that. It's all reverted now except the fix.